### ISSUE TITLE
Test compatibility with Drude trajectories

### ISSUE THREAD

**Issue by drroe** [Amber-MD/cpptraj#1098]
Make certain that cpptraj correctly identifies Drude particles.

**Comment by drroe** [comment]
`molinfo` short has weird info for the test PSF:
```
#Mol  Natom  Nres  Res0  Res1 Name C   
    1   372     2     1     2 SER  PROA     2     3     4 THR  PROA     5     5     9 GLU  PROA     2    10    11 THR  PROA     1    12    12 LYS  PROA
    2     1     1     2     2 TRP  PROA
    3     1     1     4     4 TRP  PROA
    4     1     1     9     9 TRP  PROA
    5     1     1    11    11 TRP  PROA
    6     2     1    13    13 POT  IONS
    7     2     1    14    14 POT  IONS
    8     2     1    15    15 POT  IONS
    9     2     1    16    16 POT  IONS
   10     2     1    17    17 POT  IONS
```

**Comment by drroe** [comment]
```
> molinfo short *
  [molinfo short *]
#Mol Count Natom  Nres
SER      1   372    12
TRP      4     1     1
POT      5     2     1
CLA      7     2     1
SWM4  2041     5     1
```

**Comment by drroe** [comment]
Should be addressed by #1099, #1100
